lib/modules/versioning/azure-rest-api/index.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { api as azureRestApi } from '.';
+
+describe('modules/versioning/azure-rest-api/index', () => {
+  it.each`
+    version                        | expected
+    ${'0000-00-00'}                | ${true}
```

Comment:
looks like we can rename to a simple date versioning?

Reply:
Hm, I would prefer the current name, because there are specific rules for Azure REST API versions that may not apply to simple date versioning:

- DO use a later date for each new preview version
- DO NOT use the same date when transitioning from a preview API to a GA API. If the preview api-version is '2021-06-04-preview', the GA version of the API must be a date later than 2021-06-04

Source: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#api-versioning